examples/apps/ai-collab/src/components/UserPresenceGroup.tsx

```diff
@@ -125,11 +107,11 @@ const UserPresenceGroup: React.FC<UserPresenceProps> = ({
 							<Avatar alt="User Photo" src={photo} sx={{ width: 56, height: 56 }} />
 						</StyledBadge>
 					))}
-					{photoUrls.length > 4 && (
+					{photoUrls.size > 4 && (
```

Comment:
Magic number 4: can we extract and document this?

Reply:
Sure. Adding comments here. It just looks prettier if there are more than 4 users. We will only show 4 users and a + remaining_number